charts/renku/values.yaml

```diff
@@ -494,9 +494,9 @@ ui:
   #   tag: latest
 
   ## Where to fetch default templates from
-  templatesRepository:
-    url: https://github.com/SwissDataScienceCenter/renku-project-template
-    ref: a7d01246fdbe247752a99b14c0b6e92666910b8c
+  # templatesRepository:
+  #   url: https://github.com/SwissDataScienceCenter/renku-project-template
+  #   ref: a7d01246fdbe247752a99b14c0b6e92666910b8c
```

Comment:
```suggestion
  #   ref: 0.1.3
```

Reply:
ok, I guess it makes sense to provide a tag here to be coherent with what we do in the ui chart even if the user can choose any valid reference